src/email/components/Text.tsx

```diff
@@ -26,7 +28,7 @@ export const Text = ({
 				padding="0"
 				fontSize={largeText ? '20px' : '17px'}
 				lineHeight="1.35"
-				letterSpacing="-0.02px"
+				letterSpacing={letterSpacing ? letterSpacing : '-0.02px'}
```

Comment:
Why is _all_ of our text slightly squished?

Reply:
It's because the designs called for this: https://github.com/guardian/gateway/pull/1133 (in figma file https://www.figma.com/file/ZDPonnBXbMhV2smf51Dylc/?node-id=4175%3A126191).

So yeah 🤷🏾

Reply:
Oooh https://github.com/guardian/gateway/pull/1133/files#r756130367